src/main/java/io/opentracing/contrib/global/GlobalSpanBuilder.java

```diff
@@ -0,0 +1,105 @@
+package io.opentracing.contrib.global;
+
+import io.opentracing.NoopSpanContext;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.contrib.global.delegation.DelegateSpan;
+import io.opentracing.contrib.global.delegation.DelegateSpanBuilder;
+
+import java.io.Closeable;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * SpanBuilder implementation that delegates all regular {@link Tracer.SpanBuilder} methods to the specified
+ * delegate builder instead.<br>
+ * Only the {@link #start()} method is overridden,
```

Comment:
these notes cover the what. maybe they can focus more on the why. For example, it is very interesting that start() implies a child of the current context. There are probably implications that can be explained here